drivers/clock_control/Kconfig.nrf

```diff
@@ -128,6 +128,26 @@ config CLOCK_CONTROL_NRF_USES_TEMP_SENSOR
 endif # CLOCK_CONTROL_NRF_DRIVER_CALIBRATION
 endif # CLOCK_CONTROL_NRF_K32SRC_RC_CALIBRATION
 
+config CLOCK_CONTROL_NRF_HFINT_CALIBRATION
+	bool "HFINT clock calibration"
+	depends on SOC_NRF54L05 || SOC_NRF54L10 || SOC_NRF54L15
+	help
+	  Enables periodic calibration of high-frequency clock. Calibration
+	  is experimental.
+
+if CLOCK_CONTROL_NRF_HFINT_CALIBRATION
+
+config CLOCK_CONTROL_NRF_HFINT_CALIBRATION_PERIOD
+	int "HFINT clock calibration period in milliseconds"
+	default 60000
+	help
+	  Periodically, high-frequency internal clock calibration is
+	  performed. This includes requesting high-frequency clock and
+	  starting actual calibration. Once calibration is finished,
+	  the clock is released.
```

Comment:
`Set to 0 to disable periodic calibration - in such case calibration will be performed only before HFXO is started by the application itself.`